core/ui/TiddlerInfo/References.tid

```diff
@@ -4,4 +4,4 @@ caption: {{$:/language/TiddlerInfo/References/Caption}}
 
 \define lingo-base() $:/language/TiddlerInfo/
 <$list filter="[all[current]backlinks[]sort[title]]" emptyMessage=<<lingo References/Empty>> template="$:/core/ui/ListItemTemplate">
-</$list>
+</$list>
```

Comment:
Leave a blank line at the end of the file or not? Both versions are used in source codes, not mentioned in style guide.

Reply:
Doesn't really matter for TW. ... but some external editors may have a problem.

Reply:
If you add several empty new-lines it may be a problem if you use tiddlers as templates or with transclusion.